services/api/src/schema.ts

```diff
@@ -612,6 +609,32 @@ export const userTable = pgTable("user", {
         .notNull(),
 });
 
+export const userOrganizationMappingTable = pgTable(
+    "user_organization_mapping",
```

Comment:
just `user_organization` instead of `user_organization_mapping` should be fine since it's the concatenation of two tables, denoting are a relation (mapping) table